pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -1264,7 +1265,16 @@ def _isingxx_to_cnot_rx_cnot(phi: TensorLike, wires: WiresLike, **__):
     qml.CNOT(wires=wires)
 
 
-add_decomps(IsingXX, _isingxx_to_cnot_rx_cnot)
+def _issingxx_to_ppr_resource():
+    return {resource_rep(qml.PauliRot, pauli_word="XX"): 1}
+
+
+@register_resources(_issingxx_to_ppr_resource)
+def _issingxx_to_ppr(phi: TensorLike, wires: WiresLike, **_):
```

Comment:
Typo: issing -> ising
```suggestion
def _isingxx_to_ppr(phi: TensorLike, wires: WiresLike, **_):
```